ArtemisKit/Sources/Messages/Views/SendMessageViews/SendMessageView.swift

```diff
@@ -55,6 +55,9 @@ struct SendMessageView: View {
                 viewModel.keyboardVisible = isFocused
             }
         }
+        .onChange(of: viewModel.text) {
+            viewModel.handleListFormatting(viewModel.text)
```

Comment:
```suggestion
        .onChange(of: viewModel.text) { oldValue, newValue in
            // Only call change handler if text was entered, not when text was removed
            guard newValue.count > oldValue.count else { return }
            viewModel.handleListFormatting(viewModel.text)
```

This might fix the bug that causes deletion not to work

Reply:
I test it out I think this works 🙏🏻